app/controllers/users_controller.rb

```diff
@@ -49,7 +49,7 @@ def create
 
   def update
     respond_to do |format|
-      if @user.update_attributes(user_params)
+      if @user.update_attributes(user_params.except(User.non_updateable_params))
```

Comment:
Leaving it disabled is perfect. It doesn't obsolete the validation, though: remember all form data can be forged so it's not a sufficient to rely on the `disabled` HTML attribute: it's very easy to remove the disabled flag with your DOM inspector / editor of choice. The validation will catch those forged request.

TL;DR: it's all good with the form 👍